src/material/snack-bar/simple-snack-bar.ts

```diff
@@ -6,11 +6,20 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Component, ViewEncapsulation, Inject, ChangeDetectionStrategy} from '@angular/core';
-import {MatSnackBarRef} from './snack-bar-ref';
+import {ChangeDetectionStrategy, Component, Inject, ViewEncapsulation} from '@angular/core';
 import {MAT_SNACK_BAR_DATA} from './snack-bar-config';
+import {MatSnackBarRef} from './snack-bar-ref';
 
 
+/**
+ * Internal interface for a simple snack bar component..
+ * @docs-private
+ */
+export interface MatSimpleSnackBarInterface {
```

Comment:
We generally want to avoid suffixing with "Interface"

Reply:
I'm at a loss for names. I'd like to call it `SimpleSnackBar` and let the implementations be `MatSimpleSnackBar`, but our legacy snack bar is already named `SimpleSnackBar`

Reply:
`BaseSimpleSnackBar`?